docs/DeveloperGuide.md

```diff
@@ -301,11 +464,13 @@ Use case ends.
 
 > Windows, Linux, Unix, OS-X
 
-##### Private contact detail
-
-> A contact detail that is not meant to be shared with others
-
 ## Appendix E : Product Survey
 
-{TODO: Add a summary of competing products}
+Product | Strength | Weaknesses
+-------- | :-------- | :--------
+[Wunderlist](https://www.wunderlist.com/)|<ol type="1"><li>Cloud-based<ul><li>Ability to sync tasks</li></ul></li><li>Multiple-device Usage</li><li>Data is stored on the device and syncs with cloud storage when there’s internet access<ul><li>Faster than internet based todo apps like Google Calendar</li></ul></li><li>Provides reminders</li><li>Simple user interface not too cluttered</li><li>Able to set a deadline (for dates only) for a task</li></ol>|<ol type="1"><li>Requires a lot of ‘clicks’ and fields to fill to save a task</li><li>Unable to “block” multiple slots when the exact timing of a task is uncertain</li><li>Unable to set a due time for tasks</li></ol>
```

Comment:
I believe wunderlist have much more weaknesses and strengths that are useful for the project ~ fun fact ~ I've also done wunderlist when I was a 2103 student so every year I do look forward to this product's eval ~